src/DBFTPlugin/ConsensusService.cs

```diff
@@ -632,8 +633,7 @@ private void RequestChangeView(ChangeViewReason reason)
 
         private bool ReverifyAndProcessPayload(ExtensiblePayload payload)
         {
-            if (!payload.Verify(context.Snapshot)) return false;
-            OnConsensusPayload(payload);
+            blockchain.Ask(new Blockchain.Reverify { Inventories = new IInventory[] { payload } }).Wait();
```

Comment:
You need to call `OnConsensusPayload(payload);`

Reply:
Fixed.